modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -801,46 +849,56 @@ class StreamingOutput final: public cv::gimpl::GIslandExecutable::IOutput
         GAPI_ITT_STATIC_LOCAL_HANDLE(outputs_post_hndl, "StreamingOutput::post");
```

Comment:
Sorry, could you clarify with fix you meant here? because you put comment on ```GAPI_ITT_STATIC_LOCAL_HANDLE(outputs_post_hndl, "StreamingOutput::post");``` line without modifications

Reply:
The question is why do we need all these below changes